### PR TITLE
Portfolio: mobile menu

### PR DESCRIPTION
**Pull request by khanh-devos**
- link index.html to menu.js
- setup page Menu with fixed position floating over the body.
- set up 2 functions showMenu() and hideMenu() inside menu.js. Use onclick event to activate 2 functions.
- link anchors in page Menu to the proper sections by id

### CODE REVIEW

**Review by AbelG101 — COMMENTED** [review]
# Hi @khanh-devos 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project, that doesn't have the proper linters set up for this project. This milestone requires you to use the `html-css-js` linter which is a new setup than the one you've been using so please visit the [linters config here](https://github.com/microverseinc/linters-config/tree/master/html-css-js) and update your repo accordingly.
  - After you've set up the linters you should see 5 checks (like the one below) instead of 4 checks
  image
  - This is how yours currently looks:
     image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: js/menu.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  const mobileMenu = document.querySelector('#mobile-menu');
+  const img = document.querySelector('#menu-btn img');
+  const a = document.querySelector('#header a');
+
+  mobileMenu.style.display = 'none';
+  img.src = './imgs/IconMenu.svg';
+  a.style.opacity = 1;
+}
+
+function toggleHamburgerBtn() { // eslint-disable-line no-unused-vars
```
Comment (bobb-Rob) [inline js/menu.js]: - [ ] It not an acceptable practice to disable linters in your project. Linter where set up in the first place to help you write standard code and disabling it defeats this purpose. Kindly remove the disble linter code on 21.

File: js/menu.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  const mobileMenu = document.querySelector('#mobile-menu');
+<<<<<<< HEAD
+=======
+  
+>>>>>>> 983b7bd52daae6d01d31ab223704f94463d6184f
```
Comment (bobb-Rob) [inline js/menu.js]: - [ ] The syntax you have on line 23 to 26 are not javascript syntax, hence, it is preventing your code from working as expected. After I remove them, your menu Icon click event `showMenu` and `hideMenu` started working properly. Please fix this.

Outside the above issue, you implemented the menu feature professionally, particularly that your functions are easy to follow. Well done 👏🏾

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @khanh-devos, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi  Team @khanh-devos @Kanyorok ,

Your are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- Meaningful Commit Messages :+1: 
- No Linter Errors :100: 
- Good PR Description :1st_place_medal: 

## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**